### DISCUSSION TITLE
Searching for an `Open in codesandbox` Badge

### DISCUSSION THREAD

**Discussion by BuptStEve** [codesandbox/codesandbox-client#7876] [Q&A]
Just like this, or an online logo link. Thank you~

[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/___YOUR_PATH___)

https://developer.stackblitz.com/guides/integration/open-from-github#frontmatter-title

However I only find this https://codesandbox.io/brand#logo

**Comment by linuxfandudeguy** [comment]
![68747470733a2f2f6173736574732e636f646573616e64626f782e696f2f6769746875622f627574746f6e2d656469742d6c696d652e737667](https://github.com/user-attachments/assets/2c63e504-9840-4037-b5a7-18c378a2aff8)

here ya go :3 😄
